src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -53,22 +53,35 @@
 
 namespace WinRT.Interop
 {
-    [ComImport]
-    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
-    [Guid("EECDBF0E-BAE9-4CB6-A68E-9598E1CB57BB")]
-    internal interface IWindowNative
+ 
+#if EMBED
+    internal 

```

Comment:
We should just make this always internal.  This doesn't need to be part of the public API surface.